docs/source/documentation/concepts.rst

```diff
@@ -177,3 +177,39 @@ There are two ways to run several tasks in parallel on a same organization. The
 .. TODO:: Detail vertical and horizontal scaling
 
 .. TODO:: Explain what is substra tools
+
+
+Compute plan execution
+-----------------------
+
+Read this paragraph to understand what happens during the compute plan execution and what can be done to improve the execution time.
+
+Once a compute plan is submitted to the platform, its tasks are scheduled to be executed on each organization.
+
+On each organization, Substra fetches the assets needed for the first task, builds the Docker image of the algo and creates a container with the relevant assets. The task executes and Substra saves its outputs.
+Afterwards, every task **from the same compute plan** that uses the same algo is executed in the same Docker container.
```

Comment:
This is not a always a docker container:
```suggestion
Afterwards, every task **from the same compute plan** that uses the same algo is executed in the same container.
```

Reply:
What is it then if it's not a Docker container?

Reply:
It can be CRI-O or containerd, kubernetes supports several [container runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes/)

Reply:
And it's mainly containerd that we are using !

Reply:
How do we choose between Docker and containerd? Does it depend on the task?

And the user does provide a Dockerfile for each task. So are these dockerfile then "translated" somehow to be used by other container runtimes?

Reply:
I discussed this with @AlexandrePicosson, it's much clearer now.